tests/trainer/flags/test_fast_dev_run.py

```diff
@@ -15,6 +15,7 @@ def test_skip_on_fast_dev_run_tuner(tmpdir, tuner_alg):
     """ Test that tuner algorithms are skipped if fast dev run is enabled """
 
     model = BoringModel()
+    model.lr = 0.1  # avoid no-lr-found exception
```

Comment:
pls override the `BoringModel` wherever `lr` is required

Reply:
This is fine since the attribute is set on the variable, not the class